rdtools/normalization.py

```diff
@@ -602,13 +586,16 @@ def trapz_aggregate(time_series, target_frequency, max_timedelta=None):
         max_interval_nanoseconds = max_timedelta.total_seconds() * 10.0**9
 
     # in x*hours
-    trap_sum = (values[1:] + values[:-1]) / 2 * t_diffs / 10**9 / 3600.0
+    if series_type is 'instantaneous':
+        series_sum = (values[1:] + values[:-1]) / 2 * t_diffs / 10**9 / 3600.0
+    elif series_type is 'right_labeled':
```

Comment:
Using `is` for string equality would probably work in the vast majority of cases here, but it does have some weird edge cases.  Would be safer to use `==`.  Example:

```python
In [13]: "instantaneous_"[:-1] is "instantaneous"
Out[13]: False

In [14]: "instantaneous_"[:-1] == "instantaneous"
Out[14]: True
```